lib/chat_web/live/main_live/page/lobby.ex

```diff
@@ -7,7 +7,7 @@ defmodule ChatWeb.MainLive.Page.Lobby do
   alias Phoenix.PubSub
 
   alias Chat.AdminRoom
-  alias Chat.Db.StatusPoller
+  alias Chat.Db.{StatusPoller, FreeSpaces}
```

Comment:
why do we need it for Looby?

Reply:
Because I want when db_status is changed and reassigned, free_spaces will also be reassigned. So that the liveview will real-time update when the usb is plugged in or out.

Reply:
USB plug in  has nothing to do with entering Lobby
More over, data can be written on drives constantly. Polling it once a second or 2 looks ok to make it like real-time